gapic-generator-cloud/templates/cloud/client/method/def/_lro.erb

```diff
@@ -29,15 +29,16 @@
 #     named arguments.
 <%- end -%>
 #
-# @yield [operation] Register a callback to be run when an operation is done.
-# @yieldparam operation [Google::Gax::Operation]
+# @yield [response, operation] Access the result along with the RPC operation
```

Comment:
nit: same as before, but there is where I'm worried folks might get confused (lro operation vs grpc operation) and why I think linking to the gRPC type might help clarify.

Reply:
Right. We discussed this as a breaking change. The new client method's block would be used to peek at the GRPC ActiveCall operation, and not to define the `Gax::Operation#on_done` callback. The type is defined 2 lines below.

Perhaps it would be less confusing if we prefixed `grpc` onto this argument name? So use `grpc_operation` instead of `operation`?

Reply:
Or, is there a specific value we can pass tot he proc instead of the entire GRPC ActiveCall Operation object?

Reply:
I can't think of anything better to use. The only real use case we have now is for headers (I think), but I'm not sure breaking it apart would help.

Reply:
Right, so if we only sent the headers this could be renamed `trailing_headers_callback` and send `response` and `headers` arguments.

Reply:
I don't have a strong preference either way. I think returning the operation is okay if we document it carefully. The metadata idea is fine too, but if there are other properties that we want to expose in the future we'll have to keep expanding the argument list.

Reply:
GRPC might have a wrapper object that restricts what users can change on the object, that they pass to interceptors. Would it be a good idea to use that here?

Reply:
I'm not sure, do you have an example?

Reply:
Hah! It looks like Operation is already the "view" object for ActiveCall. This is the right thing to use.